pkgs/applications/editors/standardnotes/standardNotes.nix

```diff
@@ -0,0 +1,31 @@
+with import <nixpkgs> {};
```

Comment:
You shouldn't import `<nixpkgs>` in nixpkgs.
Instead add arguments:
`{stdenv, appimage-run}:`
and then add a line in `pkgs/top-level/all-packages.nix` where you `callPackage` this file. `callPackage` will take care of providing the arguments.